docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -1,23 +1,26 @@
 [[modules-remote-clusters]]
 == Remote clusters
 
-The _remote clusters_ functionality enables you to establish unidirectional
-connections to a remote cluster. Remote clusters are required for
+Set up _remote clusters_ to establish unidirectional
+connections required for
 <<xpack-ccr,{ccr}>> and <<modules-cross-cluster-search,{ccs}>>.
 
-Remote cluster connections work by configuring a remote cluster and connecting
-to a limited number of nodes in that remote cluster. There are two modes for
-remote cluster connections: <<sniff-mode,sniff mode>> and
-<<proxy-mode,proxy mode>>.
+You configure a remote cluster and connect
+to a limited number of nodes in that remote cluster using either
+<<sniff-mode,sniff mode>> or <<proxy-mode,proxy mode>>.
 
 Communication with a remote cluster uses the <<modules-transport,transport
 layer>> to establish a number of <<long-lived-connections,long-lived>> TCP
 connections from the coordinating nodes of the local cluster to the chosen
 nodes in the remote cluster.
 
-[discrete]
+Use the <<cluster-remote-info, remote cluster info API>> to retrieve
+information about the configured remote clusters, as well as the remote nodes
+that the node is connected to.
```

Comment:
The imperative reads oddly to me here. I also don't think the clause about remote nodes is needed.
```suggestion
You can use the <<cluster-remote-info, remote cluster info API>> to get
information about registered remote clusters.
```